### ISSUE TITLE
[BUG - API Tiles] Examples DOWN because of unreachable hosts

### ISSUE THREAD

**Issue by AnthonyGlt** [iTowns/itowns#2227]
Some current example on iTowns website are down because the tile host failed to answer.

## Context
For the example `view_2d_map`:
It seems like c.tile.stamen.com is unreachable

For other examples (`3dtiles_basic`):
It seems like osm.oslandia.io is unreachable

## Steps to Reproduce (for bugs)
Try the examples on https://www.itowns-project.org/itowns/examples/index.html#3dtiles_basic

## Actual Behavior
![Screenshot from 2023-11-10 10-50-55](https://github.com/iTowns/itowns/assets/126568810/79cd7aaa-e23d-41bb-81dc-fa60acf154fa)
![Screenshot from 2023-11-10 10-51-23](https://github.com/iTowns/itowns/assets/126568810/5f7e0327-861c-49aa-96d8-f89c6f7aed3b)

## Possible Cause/Fix/Solution
Change our examples to request different hosts
To replace osm.oslandia.io, we could use the official OSM api: https://a.tile.openstreetmap.org.
It's already used in source_file_geojson_raster_2.html
But should be careful that the following rules are followed:
https://operations.osmfoundation.org/policies/tiles/

**Comment by AnthonyGlt** [comment]
osm.oslandia Host reachable again

**Comment by AnthonyGlt** [comment]
Concerning the example view_2d_map:
The "Stamen" api changed:
https://maps.stamen.com/stadia-partnership/
An account is needed and the prices are:

```
How much does it cost?
Once you have signed up for an account with Stadia, your cost is based on tile usage, meaning how many map tiles are requested through your app or website.

    If your current usage is less than 200,000 tiles / month—FREE for non-commercial use
    If your usage goes above 200,000 tiles / month or you are a commercial user—plans start at $20/month. The full pricing is [here](https://stadiamaps.com/pricing/).

You can reduce costs by upgrading to vector tiles. An in-depth guide to switching is available [here](https://docs.stadiamaps.com/guides/switching-your-maps-from-raster-to-vector-tiles/).



```

But we are using only for this example and we are using the "watercolor" and from the FAQ:
`If you are exclusively interested in Watercolor, the [tiles will continue to be available through the Smithsonian catalog](https://watercolormaps.collection.cooperhewitt.org/#12/40.7847/-73.9574)`

I guess that for now we could use the Smithsonian catalog.

(Note: The tiles are still accessible from https://tiles.stadiamaps.com/tiles/stamen_watercolor/ but it's not in the "official process of migration" to use directly this api)

**Comment by AnthonyGlt** [comment]
> Concerning the example view_2d_map: The "Stamen" api changed: https://maps.stamen.com/stadia-partnership/ An account is needed and the prices are:
> 
> ```
> How much does it cost?
> Once you have signed up for an account with Stadia, your cost is based on tile usage, meaning how many map tiles are requested through your app or website.
> 
>     If your current usage is less than 200,000 tiles / month—FREE for non-commercial use
>     If your usage goes above 200,000 tiles / month or you are a commercial user—plans start at $20/month. The full pricing is [here](https://stadiamaps.com/pricing/).
> 
> You can reduce costs by upgrading to vector tiles. An in-depth guide to switching is available [here](https://docs.stadiamaps.com/guides/switching-your-maps-from-raster-to-vector-tiles/).
> ```
> 
> But we are using only for this example and we are using the "watercolor" and from the FAQ: `If you are exclusively interested in Watercolor, the [tiles will continue to be available through the Smithsonian catalog](https://watercolormaps.collection.cooperhewitt.org/#12/40.7847/-73.9574)`
> 
> I guess that for now we could use the Smithsonian catalog.
> 
> (Note: The tiles are still accessible from https://tiles.stadiamaps.com/tiles/stamen_watercolor/ but it's not in the "official process of migration" to use directly this api)

**Comment by Desplandis** [comment]
Fixed by #2246
